### PR TITLE
Clickable post if excerpt is shown

### PR DESCRIPTION
**Pull request by jasontthai**
clickable_excerpt

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -23,6 +23,17 @@ <h3>
         </h3>
         {%- if site.show_excerpts -%}
           {{ post.excerpt }}
+          {% capture content_words %}
```
Comment (ashmaroli) [inline _layouts/home.html]: Using `{% capture %}` always results in a *string*. Wouldn't it be better if we used `{% assign .. %}` instead..?
```
{% assign content_length = post.content | number_of_words %}
{% assign excerpt_length = post.excerpt | number_of_words %}
...
```

File: _layouts/home.html [hunk 2]
```diff
@@ -23,6 +23,17 @@ <h3>
...
+            {{ post.content | number_of_words }}
+          {% endcapture %}
+
+          {% capture excerpt_words %}
+            {{ post.excerpt | number_of_words }}
+          {% endcapture %}
+
+          {% if content_words != excerpt_words  %}
+            <p><a href="{{site.baseurl}}{{ post.url }}">Read more</a></p>
```
Comment (ashmaroli) [inline _layouts/home.html]: Please prefer using Jekyll's `URLFilters` when URLs come into the picture:
```
<p><a href="{{ post.url  | relative_url }}">Read more</a></p>
```

File: _layouts/home.html (hunk 1 above)
Comment (zirius) [reply _layouts/home.html]: I figure content and excerpt are string, that's why I used `{% capture %}`, but I don't see why we shouldn't use `{% assign %}` instead. Let me update the PR.

File: _layouts/home.html (hunk 2 above)
Comment (zirius) [reply _layouts/home.html]: Updated the PR.

File: _layouts/home.html (hunk 1 above)
Comment (ashmaroli) [reply _layouts/home.html]: `content` and `excerpt` are strings, but the result from ` | number_of_words` isn't..

Comment (zirius) [reply _layouts/home.html]: Ah right!

File: _layouts/home.html [hunk 3]
```diff
@@ -23,6 +23,12 @@ <h3>
         </h3>
         {%- if site.show_excerpts -%}
           {{ post.excerpt }}
+          {% assign content_words = post.content | number_of_words %}
+          {% assign excerpt_words = post.excerpt | number_of_words %}
+
+          {% if content_words != excerpt_words  %}
+            <p><a href="{{ post.url  | relative_url }}">Read more</a></p>
```
Comment (DirtyF) [inline _layouts/home.html]: This introduces an a11y issue when all read more links will have the same label. People using Voice over or similar software, will not be able to distinguish where those links get to.

Comment (zirius) [reply _layouts/home.html]: I updated per your suggestion. Since the whole post is clickable, I have to strip the html from the excerpt too.

**Comment by ashmaroli** [comment]
@zirius Any possible side-effects from stripping HTML from rendered excerpts?

**Comment by zirius** [comment]
> @zirius Any possible side-effects from stripping HTML from rendered excerpts?

There is one side effect: html elements like links and images will be stripped and shown as regular texts.

**Comment by ashmaroli** [comment]
Thank you for proposing this enhancement @jasontthai, but I feel the added complexity is a bit too much for Minima.

**Pull request closed without merging**